src/components/MediaDevicesPreview.vue

```diff
@@ -114,32 +111,6 @@ export default {
 		}
 	},
 
-	computed: {
-
-		currentVolumeIndicatorHeight() {
-			// refs can not be accessed on the initial render, only after the
-			// component has been mounted.
-			if (!this.mounted) {
-				return 0
-			}
-
-			// WebRTC volume goes from -100 (silence) to 0 (loudest sound in the
-			// system); for the volume indicator only sounds above the threshold
-			// are taken into account.
-			let currentVolumeProportion = 0
-			if (this.currentVolume > this.volumeThreshold) {
-				currentVolumeProportion = (this.volumeThreshold - this.currentVolume) / this.volumeThreshold
-			}
-
-			const volumeIndicatorStyle = window.getComputedStyle ? getComputedStyle(this.$refs.volumeIndicator, null) : this.$refs.volumeIndicator.currentStyle
-
-			const maximumVolumeIndicatorHeight = this.$refs.volumeIndicator.parentElement.clientHeight - (parseInt(volumeIndicatorStyle.bottom, 10) * 2)
-
-			return maximumVolumeIndicatorHeight * currentVolumeProportion
-		},
-
-	},
-
 	mounted() {
 		this.mounted = true
 	},
```

Comment:
`this.mounted` property of the component is unused now. Maybe remove it from `data()` as well as this lifecycle hook? 

(The same was done in `LocalMediaControls.vue`)

Reply:
There is also no need to have `this.currentVolume` and `this.volumeThreshold` in component's `data` as they are already defined in the `devices` mixin. Maybe remove them too?